torch/autograd/variable.py

```diff
@@ -432,7 +432,11 @@ def __dir__(self):
     class _torch(object):
         @staticmethod
         def normal(means, std=1):
-            return Normal.apply(means, std)
+            if isinstance(means, Variable):
```

Comment:
I don't know how was the behavior before, but this means that one can't learn the `mean`/`std` parameters that are used to sample from a normal distribution?

Reply:
It wasn’t reparsmetrized before this PR, so you couldn’t do it anyway. The only mechanism that would let you backprop to mean and std is reinforce, which was replaced by log_prob method